web/app/components/base/chat/chat/chat-input.tsx

```diff
@@ -120,7 +120,7 @@ const ChatInput: FC<ChatInputProps> = ({
       <Send03
         className={`
           w-5 h-5 text-gray-300 group-hover:text-primary-600
-          ${!!query.trim() && 'text-primary-600'}
+          ${query.trim() ? 'text-primary-600' : ''}
```

Comment:
Some of them are using `&&`, some of them are using `? a : b`
This should start with cn as well.

Reply:
When attempting to fix the layout issue with the nav component, I discovered similar issues that this PR didn't address. Should we address them in this PR as well?